tests/unit/h/models/user_deletion_test.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+
+def test___repr__(factories, helpers):
+    user_deletion = factories.UserDeletion()
+    repr_ = repr(user_deletion)
+
+    helpers.repr_.assert_called_once_with(
+        user_deletion,
+        [
+            "id",
+            "userid",
+            "requested_at",
+            "requested_by",
+            "tag",
+            "created_at",
+            "num_annotations",
+        ],
+    )
+    assert repr_ == helpers.repr_.return_value
+
+
+@pytest.fixture(autouse=True)
+def helpers(mocker):
+    helpers = mocker.patch("h.models.user_deletion.helpers")
+    # __repr__() needs to return a string or repr() raises.
+    helpers.repr_.return_value = "test_string_representation"
+    return helpers
```

Comment:
We can either have a boilerplatey mock-based test like this, or we could use `# pragma: nocov`, or we could have a test that does not mock the helper (which sounds nicer but it actually a pain to right because the test would have to pass hard-coded values for the whole long list of model attrs, unless you want the test code to look almost the same as the `repr_()` helper code). I don't think it makes much difference :man_shrugging:

Reply:
Yeah, # pragma: nocover could have work here. This works as well of course.